api/src/certification/configuration/domain/usecases/index.js

```diff
@@ -29,6 +30,7 @@ import * as versionsRepository from '../../infrastructure/repositories/versions-
  * @typedef {learningContentRepository} LearningContentRepository
  * @typedef {skillRepository} SkillRepository
  * @typedef {tubeRepository} TubeRepository
+ * @typedef {scoBlockedAccessDatesRepository} scoBlockedAccessDatesRepository
```

Comment:
Pour respecter la convention en place (a changer aussi cote usecase du coup)

```suggestion
 * @typedef {scoBlockedAccessDatesRepository} ScoBlockedAccessDatesRepository
```

Reply:
Corrigé